definitions/variable/emissions/emissions.yaml

```diff
@@ -43,28 +46,33 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|*|AFOLU|Biomass Burning:
```

Comment:
```suggestion
- Emissions|{Level-3 Species}|AFOLU|Biomass Burning:
```

Reply:
Done in 821a8d0211a7e1c778e5dcf71a3b5f90fe8ad81b